src/redux/Graphql/saga.js

```diff
@@ -95,6 +240,10 @@ export function* getActionableTopicsHandler() {
         value: [],
         limit: action.limit,
         skip: action.skip,
+        error: {
```

Comment:
these background graphql calls dont need error objects since we fail silently, remember?